cdc/sink/metrics.go

```diff
@@ -41,11 +41,28 @@ var (
 			Name:      "execution_error",
 			Help:      "total count of execution errors",
 		}, []string{"capture", "changefeed"})
+	conflictDetectDurationHis = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: "ticdc",
+			Subsystem: "sink",
+			Name:      "conflict_detect_duration",
+			Help:      "Bucketed histogram of conflict detect time (s) for single DML statement",
+			Buckets:   prometheus.ExponentialBuckets(0.00005, 2, 21),
+		}, []string{"capture", "changefeed"})
+	bucketSizeCounter = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: "ticdc",
+			Subsystem: "sink",
+			Name:      "bucket_size",
+			Help:      "size of the DML bucket",
+		}, []string{"capture", "changefeed", "bucket"})
```

Comment:
Looks like gauge is more suitable for size.

Reply:
Because the sink worker flushes DMLs every 100ms, it is hard to track real bucket size if using gauge